common/performance.py

```
@@ -0,0 +1,721 @@
+
+
+class CachedTemplateView(TemplateView):
+    """Template view with automatic caching enabled."""
+
+    @classmethod
+    def as_view(cls, **initkwargs) -> Callable[..., HttpResponse]:
+        """
+        Returns a view function for the template view with caching enabled.
+        """
+        return cache_page(int(os.environ["CACHE_TTL"]))(
+            super(CachedTemplateView, cls).as_view(**initkwargs)
```

Ruff (0.12.2)

Use `super()` instead of `super(__class__, self)`

Remove `super()` parameters

(UP008)